persist-cli-tests/src/test/resources/test-src/input/tool_test_generate_10/persist/Persist.toml

```diff
@@ -0,0 +1,11 @@
+provider = "mysql"
+
+[database]
+host = "${persist_generate_10.clients.host}"
+port = "${persist_generate_10.clients.port}"
+user = "${persist_generate_10.clients.user}"
+password = "${persist_generate_10.clients.password}"
+database = "${persist_generate_10.clients.database}"
+
+[shadowDatabase]
+
```

Comment:
Shall we not add `shadowDatabase` for now? this is for future improvement when we support migration

Reply:
updated in [d1e9e41](https://github.com/ballerina-platform/persist-tools/pull/52/commits/d1e9e4195fff39c00f1fce312aa40487f7196262)